core/src/main/resources/hudson/PluginWrapper/PluginWrapperAdministrativeMonitor/message_pl.properties

```diff
@@ -0,0 +1,23 @@
+# The MIT License
+#
+# Copyright (c) 2004-2016, Kohsuke Kawaguchi, Sun Microsystems, Inc., and a number of other of contributors
```

Comment:
Used the most generic copyrights I've found, actually for new files it is used by default